packages/flutter/test/material/theme_test.dart

```diff
@@ -47,6 +47,68 @@ void main() {
     expect(Theme.of(tester.element(find.text('menuItem'))).brightness, equals(Brightness.dark));
   });
 
+  group('Theme.brightnessOf', () {
```

Comment:
To test the value when no MediaQuery or ThemeData is present, you can use a test like this:

```
    testWidgets('returns Brightness.light when no theme or media query is present', (
      WidgetTester tester,
    ) async {
      await tester.pumpWidget(
        const SizedBox(),
      );

      expect(Theme.brightnessOf(tester.element(find.byType(SizedBox))), equals(Brightness.light));
    });
```